pandas/tests/series/methods/test_value_counts.py

```diff
@@ -177,3 +178,28 @@ def test_value_counts_categorical_with_nan(self):
             exp = Series([2, 1, 3], index=CategoricalIndex(["a", "b", np.nan]))
             res = ser.value_counts(dropna=False, sort=False)
             tm.assert_series_equal(res, exp)
+
+    @pytest.mark.parametrize(
+        "ser, dropna, exp",
+        [
+            (
+                pd.Series([False, True, True, pd.NA]),
+                False,
+                pd.Series([2, 1, 1], index=[True, False, pd.NA]),
+            ),
+            (
+                pd.Series([False, True, True, pd.NA]),
+                True,
+                pd.Series([2, 1], index=[True, False]),
+            ),
+            (
+                pd.Series(range(3), index=[True, False, np.nan]).index,
+                False,
+                pd.Series([1, 1, 1], index=[True, False, pd.NA]),
+            ),
+        ],
+    )
+    def test_value_counts_bool_with_nan(self, ser, dropna, exp):
```

Comment:
I may have misunderstood the original issue, but my understanding is that the values were correct, just the repr was wrong. If so, why are we adding new tests here, which I think don't cover the reported issue?

Reply:
Ah, I see it was requested by @jreback in https://github.com/pandas-dev/pandas/pull/32242/files#r385098106. I suppose no harm in more tests, but don't we think this is already tested?

Reply:
@TomAugspurger Yes, I agree that this has been tested. Was following the suggestion of @jreback.

Reply:
@jreback can you comment on the request / need for these tests?